shared/secrets_masker/tests/secrets_masker/test_secrets_masker.py

```diff
@@ -355,11 +354,6 @@ def __init__(self, required_arg):
     @pytest.mark.parametrize(
         "state, expected",
         [
-            (DagRunState.SUCCESS, "success"),
-            (TaskInstanceState.FAILED, "failed"),
-            ([DagRunState.SUCCESS, DagRunState.RUNNING], ["success", "running"]),
-            ([TaskInstanceState.FAILED, TaskInstanceState.SUCCESS], ["failed", "success"]),
-            (State.failed_states, frozenset([TaskInstanceState.FAILED, TaskInstanceState.UPSTREAM_FAILED])),
```

Comment:
This was end of the day just testing enums, removing these so that `airflow` isnt imported in here